src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/AppTestBase.cs

```diff
@@ -57,8 +57,8 @@ protected ToolCommand CreateDotNetCommand() => new DotNetCommand(s_buildEnv, _te
 
     protected async Task<RunResult> RunSdkStyleApp(RunOptions options)
     {
-        string runArgs = $"{s_xharnessRunnerCommand} wasm webserver --app=. --web-server-use-default-files";
-        string workingDirectory = Path.GetFullPath(Path.Combine(FindBlazorBinFrameworkDir(options.Configuration, forPublish: options.ForPublish), ".."));
+        string runArgs = $"run -c {options.Configuration} -p:GenerateRuntimeConfigurationFiles=true";
```

Comment:
Add a `FIXME` to remove the property override when the sdk fix flows in, because setting it like this forces the value to be `true`, and the targets files can't change it. thus not testing the same thing as a user would see.